src/libs/collect.c

```diff
@@ -1529,7 +1530,7 @@ static void list_view(dt_lib_collect_rule_t *dr)
           {
             orders = dt_util_dstrcat(orders, "WHEN mo.version = %d THEN '%s' ", i, _(dt_iop_order_string(i)));
           }
-          orders = dt_util_dstrcat(orders, "ELSE '%s' ", _("none"));
+          orders = dt_util_dstrcat(orders, "ELSE '%s' ", _("not altered"));
```

Comment:
I prefer just "none" here. No strong opinion, but this seems clear enough and is less verbose.